mapie/estimators.py

```diff
@@ -58,6 +59,12 @@ class MapieRegressor(BaseEstimator, RegressorMixin):  # type: ignore
 
         By default ``None``.
 
+    n_jobs: int, optional
+        Number of jobs to run in parallel.
+        -1 means using all processors.
+
+        By default 1.
```

Comment:
By default None. And None means 1.

Cf https://github.com/scikit-learn/scikit-learn/blob/15a949460/sklearn/calibration.py#L108

Reply:
C'est ce que j'avais fait pour suivre ce qui est fait dans scikit-learn, mais il faut définir comme `Optional[Union[int, None]]` pour passer le problème de `typing`. Ça te va comme ça?

Reply:
Très bien

Reply:
Ah oui bien vu, je me complique la vie pour rien :-)